core/domain/collection_domain_test.py

```diff
@@ -578,6 +578,52 @@ def test_delete_skill(self):
         self.assertEqual(
             collection.get_node('exp_id_1').prerequisite_skill_ids, [])
 
+    def test_get_skill_id_from_skill_name(self):
```

Comment:
Good tests, but they should also cover the various error cases.

Reply:
Updated. Assuming you mean exceptions.